plugins/UM3NetworkPrinting/src/Network/ZeroConfClient.py

```diff
@@ -152,6 +154,16 @@ def _onServiceAdded(self, zero_conf: Zeroconf, service_type: str, name: str) ->
 
         return True
 
+    @staticmethod
+    def _getAddress(info: ServiceInfo) -> str:
+        """Retrieve IPv4 address from the ServiceInfo as string, None if there is no address."""
+
+        # Check the version of python-zeroconf
+        if Version(zeroconf_version) >= Version([0, 24, 0]):
+            return next(iter(info.addresses), None)
```

Comment:
Next expects a string, not None

Reply:
None is the default value here, in case info.addresses is empty. As far as I know it accepts any type.